algoliasearch-core/src/main/java/com/algolia/search/models/common/InnerQuery.java

```diff
@@ -0,0 +1,79 @@
+package com.algolia.search.models.common;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.io.Serializable;
+import java.util.Objects;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class InnerQuery implements Serializable {
+
+  private String indexName;
```

Comment:
Thanks for your contribution!

This endpoint is a bit particular because all responses fields are snake cased. See the [C# Poco](https://github.com/algolia/algoliasearch-client-csharp/blob/b9c57630a3e83740f7a8b43246b37a6ac1a8ca16/src/Algolia.Search/Models/Common/LogResponse.cs#L44-L68). I discovered our [public doc](https://www.algolia.com/doc/api-reference/api-methods/get-logs/#method-response-inner_queries) is incorrect on the casing as well. :(

Reply:
Great catch! the public doc misled me :(